setup.py

```diff
@@ -25,65 +25,67 @@ def load_version():
 os.chdir(os.path.dirname(os.path.abspath(__file__)))
 
 _VERSION_GLOBALS = load_version()
-DISTNAME = 'nilearn'
-DESCRIPTION = 'Statistical learning for neuroimaging in Python'
-with open('README.rst') as fp:
+DISTNAME = "nilearn"
+DESCRIPTION = "Statistical learning for neuroimaging in Python"
+with open("README.rst") as fp:
     LONG_DESCRIPTION = fp.read()
-MAINTAINER = 'Gael Varoquaux'
-MAINTAINER_EMAIL = 'gael.varoquaux@normalesup.org'
-URL = 'https://nilearn.github.io'
-LICENSE = 'new BSD'
-DOWNLOAD_URL = 'https://nilearn.github.io'
-VERSION = _VERSION_GLOBALS['__version__']
+MAINTAINER = "Gael Varoquaux"
+MAINTAINER_EMAIL = "gael.varoquaux@normalesup.org"
```

Comment:
Since Gael is no longer involved in Nilearn, and I am the one with a permanent position, we probably need to put my coords here

Reply:
Good point.
Used the email from your web site.